pkg/beacon/relay/registry/groups_test.go

```diff
@@ -197,6 +221,12 @@ func (mgri *mockGroupRegistrationInterface) OnGroupRegistered(
 	panic("not implemented")
 }
 
+func (mgri *mockGroupRegistrationInterface) IsGroupRegistered(
```

Comment:
Is it used anywhere?

Reply:
Yep, in `TestUnregisterStaleGroups` and `TestUnregisterStaleGroupsSkipLastGroupCheck`